### PR TITLE
Add --count for list operations.

### PR DESCRIPTION
**Pull request by rabbah**
## Related Issue

#190.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by meryllblanchet** [comment]
Thanks for the submission, @rabbah ! Stay tuned for the review.

File: test/commands/runtime/action/list.test.js [hunk 1]
```diff
@@ -175,5 +175,32 @@ describe('instance methods', () => {
           expect(stdout.output).toMatchFixture('action/list-name-sort-output.txt')
         })
     })
+
+    test('return action count', () => {
```
Comment (sandeep-paliwal) [inline test/commands/runtime/action/list.test.js:179]: maybe one more test case for count = 0.
same for all other tests for package, activations etc.

Comment (rabbah) [reply test/commands/runtime/action/list.test.js:188]: I pushed a comment to add this test. Thanks @sandeep-paliwal.

**Review by sandeep-paliwal — APPROVED** [review]
LGTM
verified new flags from CLI, things working as expected.

**Pull request merged**